src/components/TitleHashButton.tsx

```diff
@@ -135,13 +136,24 @@ function Name({address, isValidator}: {address: string; isValidator: boolean}) {
           padding: "0.15rem 1rem 0.15rem 1rem",
         }}
       >
-        <Link
-          href={`https://www.aptosnames.com/name/${name}`}
-          target="_blank"
-          underline="none"
-        >
-          {`${name}.apt`}
-        </Link>
+        {name.endsWith(".apt") ? (
+          <Link
+            href={`https://www.aptosnames.com/name/${name}`}
+            target="_blank"
+            underline="none"
+          >
+            {name}
+          </Link>
+        ) : (
+          <>
+            <Typography>
```

Comment:
When working with things in a line, `flex` is your friend. Relying on white space can cause issues like bellow (based on the size of the other elements in the container).

<img width="175" alt="image" src="https://github.com/user-attachments/assets/ac5f9b5b-a633-4876-85f0-daaadc5365de">

Instead, try the following:

```tsx
  <Typography sx={{display: 'flex', alignItems: 'row', gap: 4}}>
    <span>{name}</span>
    <Tooltip title={'This is a verified address label.'}>
      <VerifiedOutlined fontSize="small" />
    </Tooltip>
  </Typography>
```

Wrapping the `name` in a span gives `flex` a node to latch on to. `alignItems` aligns on the secondary axis (in this case the y axis) which centers everything. If that still doesn't line up nicely, you have to modify the heights of the children of the flex container.